tests/test_main_notruncate.py

```diff
@@ -0,0 +1,52 @@
+# https://github.com/con2/emrichen/issues/54
+
+from pathlib import Path
+from tempfile import TemporaryDirectory
+
+import yaml
+
+from emrichen.__main__ import main
+
+ORIGINAL_YAML = """
+a: 1
+b: 2
+"""
+
+UPDATE_YAML = """
+b: 3
+"""
+
+TEMPLATE_YAML = """
+!Merge
+  - !Include original.yaml
+  - !Include update.yaml
+"""
+
+
+def test_main_notruncate():
+    with TemporaryDirectory() as temp_dir:
+        temp_path = Path(temp_dir)
+        original_path = temp_path / "original.yaml"
+        update_path = temp_path / "update.yaml"
+        template_path = temp_path / "template.yaml"
+
+        for path, content in [
+            (original_path, ORIGINAL_YAML),
+            (update_path, UPDATE_YAML),
+            (template_path, TEMPLATE_YAML),
+        ]:
+            with open(path, 'w', encoding="UTF-8") as output_file:
+                output_file.write(content)
```

Comment:
Okay, fine, let's split the `tmpdir` difference and use `tmp_path`:
```suggestion
def test_main_notruncate(tmp_path: pathlib.Path):
    original_path = tmp_path / "original.yaml"
    update_path = tmp_path / "update.yaml"
    template_path = tmp_path / "template.yaml"

    for path, content in [
        (original_path, ORIGINAL_YAML),
        (update_path, UPDATE_YAML),
        (template_path, TEMPLATE_YAML),
    ]:
        path.write_text(content, encoding="UTF-8")
```